.github/workflows/main.yml

```diff
@@ -108,6 +108,8 @@ jobs:
   unit-test:
     name: Unit Test
     runs-on: ${{ matrix.os }}
+    env:
+      CI: true
```

Comment:
The `CI` definitions should be unnecessary, the `CI` environment variable is [set by default in GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) (and most other CI system).

Reply:
Even better!